### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by Cwiiis** [comment]
@kdavis-mozilla @andrenatal This PR merges upstream, which fixes the reported issues we currently have with vaani.

File: lib/git-auto-updater.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ function scheduleUpdateCheck(error) {
     var minutes = now.getMinutes();
 
     if (hours > conf.time.hours ||
-        (minutes > conf.time.minutes && hours === conf.time.hours)) {
+        (minutes >= conf.time.minutes && hours === conf.time.hours)) {
```
Comment (kdavis-mozilla) [inline lib/git-auto-updater.js:239]: Nice!

**Comment by kdavis-mozilla** [comment]
LGTM

File: lib/git-auto-updater.js (hunk 1 above)
Comment (Cwiiis) [reply lib/git-auto-updater.js:239]: Naming things, cache invalidation and off-by-one... Definitely the hardest problems ;)

**Pull request merged**